model/src/main/proto/javaclasses/exlibris/c/commands.proto

```diff
@@ -34,105 +34,105 @@ import "google/protobuf/timestamp.proto";
 import "javaclasses/exlibris/identifiers.proto";
 import "javaclasses/exlibris/values.proto";
 
-// An attempt to create the new book.
+// An attempt to create a new book.
 //
 // In case of a successful command propagation, the book is added.
-// not be applied to the existing book.
+// Cannot be applied to the existing book.
 //
 message AddBook {
 
-    // An identifier of the book to create.
+    // The identifier of a book to create.
     BookId book_id = 1;
 
-    // Details of the new book.
+    // Details of a new book.
     BookDetails book_details = 2;
 }
 
-// An attempt to update the book.
+// An attempt to update a book.
 //
 // In case of a successful command propagation, the book is updated.
 // Cannot be applied to the missing book.
 //
 message UpdateBook {
 
-    // An identifier of the book to update.
+    // The identifier of a book to update.
     BookId book_id = 1;
 
-    // New details for the existed book.
+    // New details for an existed book.
     BookDetails book_details = 2;
 }
 
-// An attempt to remove the book.
+// An attempt to remove a book.
 //
 // In case of a successful command propagation, the book is removed.
 // Cannot be applied to the missing book.
 //
 message RemoveBook {
 
-    // An identifier of a book to remove.
+    // The identifier of a book to remove.
     BookId bok_id = 1;
 
     // The reason of removing book.
     //
-    // Book could be remove with one of the next reason:
-    // 1. Book was lost.
-    // 2. Book was outdated.
-    // 3. Custom reason that is described by librarian.
+    // A book could be removed with one of following reasons:
+    // 1. The book is lost.
+    // 2. The book is outdated.
+    // 3. Custom reason that is described by a librarian.
     oneof book_removal_reason {
         bool lost = 2;
         bool outdated = 3;
         string custom_reason = 4;
     }
 
-    // Time when book was removed.
+    // The time when a book was removed.
     google.protobuf.Timestamp when_removed = 5;
 }
 
-// Append the book inventory.
+// Append a book inventory.
 //
 // In case of a successful command propagation,
```

Comment:
We haven't rejection for this command. In my opinion "In case of a successful command propagation" is excess.